docs/blueprints/storage-node-downtime-disqualification.md

```diff
@@ -0,0 +1,112 @@
+An earlier iteration of this document suggested a fixed suspension period, i.e. once suspended, even if the node fixed its downtime issues, it could not be reinstated until the end of the suspension. This was motivated by the idea that continuous evalution of a suspended node's downtime could result in it fluctuating between suspension and reinstatement, thus never remaining in suspension long enough to become disqualified. However, we decided that is is better for the nwtwork as a whole if we allow nodes which have fixed their issues to come back ASAP. However, in order to remedy the potential for a node to avoid disqualification by fluctuating between suspension and reinstatement, we still review the node's downtime at the end of some period.
+
+### Tracking period
+One might think to measure downtime within a static window, such as within month boundaries, after which downtime is reset. This introduces a problem. If one day of downtime within the tracking period is allowed, why _not_ shut down on the final day? This is compounded by the fact that any number of nodes could have the same idea, resulting in a potentially significant portion the network going offline at the same time. The trailing tracking period does not fall victim to the same problem. There is no specific timeframe which has a higher incentive for downtime, and any observed downtime will follow the node for the length of the tracking period.
+
+### Revert to using audits for uptime reputation?
+This blueprint for disqualification was written to work with the currently implemented downtime tracking system. However, questions have recently been raised as to whether the change from using audits to check downtime to the current 'downtime tracking' system was necessary. The impetus for the transition was that a node could become disqualified by being offline for only a short period of time due to the chance that it could be audited many times in quick succession. That said, auditing does give us some indication of node downtime. Is implementing a new system for tracking downtime worth the additional resources if the audit system is already randomly reaching out to nodes? Is it possible for us to salvage the previous system and account for its flaws? Perhaps. Here is a quick idea for discussion, the details of which would require further investigation:
+
+Scale the _weight_ of the uptime check based on time elapsed since last contact.
+
+The previous uptime reputation system worked much like the current audit reputation. See [Node Selection](node-selection.md) for more information.
+> α(n) = λ·α(n-1) + _w_(1+_v_)/2
+>
+> β(n) = λ·β(n-1) + _w_(1-_v_)/2
+>
+> R(n) = α(n) / (α(n) + β(n))
+
+In the case of a successful uptime check, _v_ is set to 1. Otherwise, it is set to -1. Previously, _w_, or _weight_, was configured at runtime and thereafter remained static. Thus, all uptime checks carried the same weight. However, should this really be the case?
```

Comment:
Can you clarify why you're suggesting changing _w_ and not _v_?  (I don't remember them well enough to know, but it'd be helpful to know if you know.  :D :P